siliconcompiler/tools/openroad/openroad.py

```diff
@@ -234,7 +234,7 @@ def setup(chip, mode='batch'):
     for libvar, openroadvar in [('openroad_pdngen', 'pdn_config'),
                                 ('openroad_global_connect', 'global_connect')]:
         if chip.valid('tool', tool, 'task', task, 'var', openroadvar) and \
-           not chip.get('tool', tool, 'task', task, 'var', openroadvar, step=step, index=index):
+           chip.get('tool', tool, 'task', task, 'var', openroadvar, step=step, index=index):
```

Comment:
Could we have caught this with a test?

Also, should we have a place to record test cases which we believe could be helpful, but may not have time to implement immediately?

Reply:
@WRansohoff I'd say open a JIRA for them. I think once we fill out our CI with more actual designs these things would be easier to catch.